graylog-storage-opensearch2/src/main/java/org/graylog/storage/opensearch2/IndexToolsAdapterOS2.java

```diff
@@ -115,7 +114,7 @@ public Map<DateTime, Map<String, Long>> fieldHistogram(String fieldName, Set<Str
     @Override
     public long count(Set<String> indices, Optional<Set<String>> includedStreams) {
         final CountRequest request = new CountRequest(indices.toArray(new String[0]), buildStreamIdFilter(includedStreams))
-                .indicesOptions(IndicesOptions.fromOptions(true, false, true, false));
+                .indicesOptions(IndicesOptions.fromOptions(true, true, true, false));
```

Comment:
The same as here: https://github.com/Graylog2/graylog2-server/pull/18146/files#r1479887434

Reply:
done